Mobl_ios_Callisto-master/MealTrackingPilot/LogCommonMeal/Views/OccasionPicker.swift

```diff
@@ -693,93 +576,90 @@ fileprivate extension CGPoint {
 }
 
 fileprivate extension CGSize {
```

Comment:
Create a separate for CGSize extension line CGSize+Extensions & move the code on file.

Reply:
Done